### DISCUSSION TITLE
How to compile in Windows 10

### DISCUSSION THREAD

**Discussion by JiffyChen** [os-fpga/FOEDAG#771] [Q&A] [answered]
How to compile in Windows 10？

env:
Win10 / VS2019/qt 5.15.2/ cmake 3.25

error:
\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: “E:\FOEDAG\build\CMakeFiles\0895e43a28cf5b18a47324b3adda7f3c\tcl86ts.lib.rule;E:\FOEDAG\build\CMakeFiles\2bcb5e17a8495c2f98ac76bf8105742b\tcl_build.rule”

**Comment by skyler-rs** [comment, marked as answer]
Try the MSYS2 steps https://foedag.readthedocs.io/en/latest/tutorials/getting_started/dependencies/#msys2-mingw64 . I was able to successfully build this morning.

**Reply by JiffyChen** [reply]
thanks
